reference/outlook/preview/SeriesTime.md

```diff
@@ -0,0 +1,391 @@
+	var recurrence = asyncResult.value;
+	var endDate = recurrence.seriesTime.getEndTime();
+}
+```
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+#### getStartDate() → {String}
+
+Gets the start date of a recurrence pattern in the following [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) date format:
+"YYYY-MM-DD"
+
+##### Examples
+
+This example gets the start date of a recurrence.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+function callback(asyncResult){
+	var context = asyncResult.context;
+	var recurrence = asyncResult.value;
+	var endDate = recurrence.seriesTime.getStartDate();
+}
+```
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+#### getStartTime() → {String}
+
+Gets the start time of the meeting/appointment of a recurrence pattern in whichever time zone that the user/add-in set the recurrence pattern in the following ISO 8601 format:
+"THH:mm:ss:mmm"
+
+##### Examples
+
+This example gets the start time of a recurrence.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+function callback(asyncResult){
+	var context = asyncResult.context;
+	var recurrence = asyncResult.value;
+	var endDate = recurrence.seriesTime.getStartTime();
+}
+```
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+#### setDuration(int minutes)
+
+Sets the duration of a meeting/appointment of a recurrence pattern. This should also change the end time of the recurrence pattern.
+
+##### Parameters:
+
+|Name|Type|Attributes|Description|
+|---|---|---|---|
+|`minutes`|Integer||The length of the meeting or appointment in minutes.|
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadWriteItem|
+|Applicable Outlook mode|Compose|
+
+##### Examples
+
+This example sets the duration of a recurrence to 2 hours.
+
+```js
+var seriesTimeObject = new seriesTime()
+seriesTimeObject.setDuration(120);
+```
+
+#### setEndDate(int year, int month, int day)
+
+Sets the end date of a recurrence pattern.
```

Comment:
Should we be more clear with these descriptions? Say something like "Sets the date for when the recurring meeting/appointment series will end"